alg/gdalwarper.cpp

```diff
@@ -1167,6 +1167,17 @@ CPLErr GDALWarpDstAlphaMasker(void *pMaskFuncArg, int nBandCount,
  * will be selected, not just those whose center point falls within the
  * polygon.</li>
  *
+ * <li>XSCALE: Ratio expressing the resampling factor along the target
+ * horizontal axis. Equals to one for no resampling, below one for downsampling
+ * and above one for upsampling. This is automatically computed, for each
```

Comment:
Add something like "The scale is used to determine the number of source pixels along the x-axis that are considered by the resampling algorithm." ?

(Does it have another purpose?)

Reply:
applied, thanks